README.md

```diff
@@ -134,6 +135,25 @@ where
 
     Value associated with ANY key MAY be NULL.
 
+#### HW Capability Request
+
+* HW_CAP/'msg-correlation-id'/'type'
+
+where:
+
+* 'type' can be GPI, GPO, serial (not supported yet)
```

Comment:
is this case insensitive? otherwise 'gpi' and 'gpo' are more suitable

Reply:
well, fty-sensor-gpio uses capital letters too

Reply:
ok, this is part of the the message it should not be capital. Corrected